common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -45,7 +43,7 @@ def _set_up_course_xml(cls):
         """
         Sets up and returns XML course structure.
         """
-        course = xml.CourseFactory.build()
+        course = xml.CourseFactory.build(end=str(TOMORROW))
```

Comment:
I'll update with `FUTURE_DATE` and `PAST_FUTURE_DATE`, and define those to be a week or more in the future.